### PR TITLE
JAV-473 fix usage of HttpServerFilter.afterReceiceRequest result

### PR DESCRIPTION
**Pull request by wujimin**
when HttpServerFilter.afterReceiceRequest return none null, do not call into business operation

### CODE REVIEW

File: common/common-rest/src/test/java/io/servicecomb/common/rest/TestAbstractRestInvocation.java [hunk 1]
```diff
@@ -202,6 +202,7 @@ public void invokeFilterHaveResponse(@Mocked HttpServerFilter filter) {
     restInvocation = new AbstractRestInvocation() {
       @Override
       protected void doInvoke() throws Throwable {
+        result.value = Response.ok("not run to here");
```
Comment (eric-lee-ltk) [inline common/common-rest/src/test/java/io/servicecomb/common/rest/TestAbstractRestInvocation.java]: how about change to "This should not run" or "expect not to run here"?

File: edge/edge-core/src/test/java/io/servicecomb/edge/core/TestEdgeInvocation.java [hunk 2]
```diff
@@ -275,6 +279,45 @@ public void invokeNormal(@Mocked MicroserviceVersionMeta latestMicroserviceVersi
...
+      }
+
+      @Override
+      protected void sendResponse(Response response) throws Exception {
+        result.value = response;
+      }
+
+      @Override
+      protected void doInvoke() throws Throwable {
+        result.value = Response.ok("do not run to here");
```
Comment (eric-lee-ltk) [inline edge/edge-core/src/test/java/io/servicecomb/edge/core/TestEdgeInvocation.java]: same as above

File: common/common-rest/src/test/java/io/servicecomb/common/rest/TestAbstractRestInvocation.java (hunk 1 above)
Comment (wujimin) [reply common/common-rest/src/test/java/io/servicecomb/common/rest/TestAbstractRestInvocation.java]: i feet that this is not necessary

File: edge/edge-core/src/test/java/io/servicecomb/edge/core/TestEdgeInvocation.java (hunk 2 above)
Comment (wujimin) [reply edge/edge-core/src/test/java/io/servicecomb/edge/core/TestEdgeInvocation.java]: same as above

**Pull request merged**